e2e/test/scenarios/question-reproductions/reproductions-3.cy.spec.js

```diff
@@ -566,7 +566,7 @@ describe("issue 40435", () => {
     queryBuilderHeader().findByText("Save").click();
     modal().last().findByText("Save").click();
     cy.wait("@updateCard");
-    visualize();
+    cy.findByLabelText("Switch to visualization").click();
```

Comment:
Design feedback: when using table viz it's not easy to tell which button does what, as the icons are identical:

cc: @AlessioLaiso 

![image](https://github.com/user-attachments/assets/a812c804-c55f-4672-88c5-b84f0e75ad6c)

Reply:
it's a request from the previous PR, tooltip shows the correct one. You'll see two table icons only with table and pivot table visualization types